docs/acme-divergences.md

```diff
@@ -4,61 +4,61 @@ While Boulder attempts to implement the ACME specification as strictly as possib
 
 Boulder doesn't return errors under the `urn:ietf:params:acme:error:` namespace but instead uses the `urn:acme:error:` namespace from [draft-ietf-acme-01 Section 5.4](https://tools.ietf.org/html/draft-ietf-acme-acme-01#section-5.4).
 
 Boulder uses `invalidEmail` in place of the error `invalidContact` defined in [draft-ietf-acme-01 Section 5.4](https://tools.ietf.org/html/draft-ietf-acme-acme-01#section-5.4).
 
 Boulder does not implement the `caa` and `dnssec` errors.
 
-## [Section 6.1.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.1)
+## [Section 6.1.](https://tools.ietf.org/html/draft-ietf-acme-acme-04#section-6.1)
 
 Boulder does not implement the `new-application` resource. Instead of `new-application` Boulder implements the `new-cert` resource that is defined in [draft-ietf-acme-02 Section 6.5](https://tools.ietf.org/html/draft-ietf-acme-acme-02#section-6.5).
 
-## [Section 6.1.1.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.1.1)
+## [Section 6.1.1.](https://tools.ietf.org/html/draft-ietf-acme-acme-04#section-6.1.1)
 
 Boulder does not implement the `meta` field returned by the `directory` endpoint.
 
-## [Section 6.1.2.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.1.2)
+## [Section 6.1.2.](https://tools.ietf.org/html/draft-ietf-acme-acme-04#section-6.1.2)
 
-Boulder does not implement the `status`, `applications`, or `certificates` fields
+Boulder does not implement the `terms-of-service-agreed`, `applications`, or `certificates` fields
 in the registration object (nor the endpoints the latter two link to).
 
-## [Section 6.1.3.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.1.3)
+## [Section 6.1.3.](https://tools.ietf.org/html/draft-ietf-acme-acme-04#section-6.1.3)
 
 Boulder does not implement applications, instead it implements the `new-cert` flow from [draft-ietf-acme-02 Section 6.5](https://tools.ietf.org/html/draft-ietf-acme-acme-02#section-6.5). Instead of application requirements Boulder currently uses authorizations that are created using the `new-authz` flow from [draft-ietf-acme-02 Section 6.4](https://tools.ietf.org/html/draft-ietf-acme-acme-02#section-6.4).
 
-## [Section 6.1.4.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.1.4)
+## [Section 6.1.4.](https://tools.ietf.org/html/draft-ietf-acme-acme-04#section-6.1.4)
 
 Boulder does not implement the `scope` field in authorization objects.
 
-## [Section 6.2.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.2)
+## [Section 6.2.](https://tools.ietf.org/html/draft-ietf-acme-acme-04#section-6.3)
 
-Boulder does not allow `tel` URIs in the registrations `contact` list.
+Boulder only allows `mailto` URIs in the registrations `contact` list.
 
-## [Section 6.2.1.](https://tools.ietf.org/html/draft-ietf-acme-acme-03#section-6.2.1)
+## [Section 6.2.1.](https://tools.ietf.org/html/draft-ietf-acme-acme-04#section-6.3.1)
```

Comment:
Oops, also looks like this should be linking to Section 6.3.2 for Account Key Rollover. 6.3.1 is "Change of Terms of Service" in 04.